java/src/jmri/jmrit/beantable/BeanTableDataModel.java

```diff
@@ -227,8 +254,17 @@ public Object getValueAt(int row, int col) {
             case DELETECOL:  //
                 return Bundle.getMessage("ButtonDelete");
             default:
-                log.error("internal state inconsistent with table requst for {} {}", row, col);
-                return null;
+                NamedBeanPropertyDescriptor desc = getPropertyColumnDescriptor(col);
+                if (desc == null) {
+                    log.error("internal state inconsistent with table requst for {} {}", row, col);
+                    return null;
+                }
+                b = getBySystemName(sysNameList.get(row));
+                String s = (String)b.getProperty(desc.propertyKey);
```

Comment:
Not all results of NamedBean.getProperty(key) are Strings, so its better to return whatever the actual object is instead of coercing it to a String at this point, and return the value of S in line 267.

Reply:
done